hypertools/tools/cluster.py

```diff
@@ -1,11 +1,15 @@
 #!/usr/bin/env python
 
 from sklearn.cluster import KMeans
-from sklearn.decomposition import PCA
+import warnings
 import numpy as np
 from .._shared.helpers import *
+from .reduce import reduce as reducer
+from .align import align as aligner
+from .normalize import normalize as normalizer
 
-def cluster(x, n_clusters=8, ndims=None):
+def cluster(x, n_clusters=8, ndims=None, model='IncrementalPCA',
```

Comment:
@jeremymanning - for the `plot` function (as well as `reduce` and `align`), we use the keyword `model` to specify the dimensionality reduction model.  for the `align` and `cluster` functions, this may be a little weird since you might expect `model` to refer to the alignment/clustering model respectively, instead of the dimensionality reduction model.  Do you have an opinion on how to handle this?

If we want to support different alignment/clustering models in the future, we could add a `align_model` and `cluster_model` flag.  We could also add a `reduce_model` flag that could do the same thing as `model`.